spec/requests/workflows/destroy_all_spec.rb

```diff
@@ -11,11 +11,12 @@
     let!(:item2) do
       FactoryBot.create(:workflow_step,
                         repository: 'sdr',
+                        process: 'publish',
                         druid: druid)
     end
 
     let!(:item3) do
-      FactoryBot.create(:workflow_step)
+      FactoryBot.create(:workflow_step, process: 'shelve')
```

Comment:
necessary for tests to continue to work with our new validation to prevent dupe steps from being added